R/navbar.R

```diff
@@ -92,9 +96,6 @@ navbar_articles <- function(pkg = ".") {
     menu$intro <- menu_link("Get started", intro$file_out)
   }
 
-  if (!is.null(pkg$repo$url$home)) {
```

Comment:
I'm not sure why this was in `navbar_articles()`

Reply:
Ooops!